specification/managementgroups/resource-manager/Microsoft.Management/preview/2023-12-01-alpha/management.json

```diff
@@ -2,9 +2,9 @@
   "swagger": "2.0",
   "host": "management.azure.com",
   "info": {
-    "version": "2023-04-01",
-    "title": "Management Groups API",
-    "description": "The Azure Management Groups API enables consolidation of multiple \nsubscriptions/resources into an organizational hierarchy and centrally \nmanage access control, policies, alerting and reporting for those resources.\n"
+    "version": "2023-12-01-alpha",
```

Comment:
why not use "2023-12-01-preview" ?

Reply:
Since the APIs are resource types are being developed as a new concept, we thought it would be better to go with alpha ( expecting some bugs ) and have preview version to be more stable than now. 
Is the usual suggestion is to go with preview version from the start ?

Reply:
I believe current guidance is to use YYYY-MM-DD-preview for all pre-stable APIs.